modules/local/report.nf

```diff
@@ -32,7 +32,7 @@ process REPORT {
     def genelist_opt = genelist.name != 'NO_FILE' ? "--genelist $genelist" : ''
     def relevel_opt = relevel.name != 'NO_FILE2' ? "--relevel $relevel" : ''
     def batch_effect_opt = params.batch_effect ? "--batch_effect TRUE" : ''
-    def rlog_opt = params.use_vst ? '--rlog FALSE' : ''
+//    def rlog_opt = params.use_vst ? '--rlog FALSE' : ''
```

Comment:
Remove or comment in?

Reply:
Örks. Good catch, I completely forgot to remove it